### PR TITLE
fix(snippet): snippet body's size is too long to insert into metadb

### LINKED ISSUE #855: [Bug]: Failed to create code snippet when the number of characters in the code snippet is close to 100000
### ODC version

odc423

### OB version

*

### What happened?

02decc44572681105e1c6466e412ae31
5e8122aef84281d0d5e5dcdbab169e11

### What did you expect to happen?

Created successfully

### How can we reproduce it (as minimally and precisely as possible)?

Create a code snippet with a character count close to 100000

### Anything else we need to know?

_No response_

### Cloud

_No response_

### PR DESCRIPTION
**Pull request by yhilmare**
#### What type of PR is this?

type-bug

#### What this PR does / why we need it:

the data type of snippet body's corresponding metadb column is text which is limited to 65535. An exception will be thrown by metadb when you attempt to insert a too long data.

This pr add a constraint to the snippet body~

#### Which issue(s) this PR fixes:

Fixes #855

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/SqlParser.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import com.oceanbase.tools.dbbrowser.parser.result.ParseSqlResult;
 import com.oceanbase.tools.sqlparser.obmysql.OBLexer;
 import com.oceanbase.tools.sqlparser.obmysql.OBParser;
+import com.oceanbase.tools.sqlparser.util.TimeoutTokenStream;
```
Comment (smallsheeeep) [inline libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/SqlParser.java:30]: same as above

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/PLParser.java [hunk 2]
```diff
@@ -33,6 +33,7 @@
 import com.oceanbase.tools.sqlparser.oracle.PlSqlLexer;
 import com.oceanbase.tools.sqlparser.oracle.PlSqlParser;
 import com.oceanbase.tools.sqlparser.util.CaseChangingCharStream;
+import com.oceanbase.tools.sqlparser.util.TimeoutTokenStream;
```
Comment (smallsheeeep) [inline libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/PLParser.java:36]: import but no usage?

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/SqlParser.java (hunk 1 above)
Comment (yhilmare) [reply libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/parser/SqlParser.java:30]: TimeoutTokenStream is defined in db-browser originally and I move it to the ob-sql-parser, to that the import is changed~

**Review by smallsheeeep — APPROVED** [review]
lgtm

**Pull request merged**